pkg/kube/pvc.go

```diff
@@ -68,10 +68,9 @@ func (p *pvcClient) GetPVC(namespace, name string) (*corev1.PersistentVolumeClai
 
 func (p *pvcClient) ListPVCs(namespace string, selector labels.Selector) ([]corev1.PersistentVolumeClaim, error) {
 	pvcList := &corev1.PersistentVolumeClaimList{}
-	if err := p.kubecli.List(context.TODO(), pvcList, &client.ListOptions{LabelSelector: selector, Namespace: namespace}); err != nil {
-		if err != nil {
-			return nil, err
-		}
+	err := p.kubecli.List(context.TODO(), pvcList, &client.ListOptions{LabelSelector: selector, Namespace: namespace})
+	if err != nil {
+		return nil, err
```

Comment:
When I was going to change the style, I saw the code from L90-L98, which didn't follow the Uber style.

I wonder again if it is necessary to follow the style at the cost of readability.

@veezhang

Reply:
Thanks! Somewhere are not strictly following the style, we will improve the code quality.